api_shield.go

```diff
@@ -68,3 +70,15 @@ func (api *API) UpdateAPIShieldConfiguration(ctx context.Context, rc *ResourceCo
 
 	return asResponse, nil
 }
+
+func validateAPIShieldConfigurationUpdateParams(params UpdateAPIShieldParams) UpdateAPIShieldParams {
+	// edge case for when we get an empty auth_id_characteristics. We want to send `{auth_id_characteristics: []}` to the API.
```

Comment:
```suggestion
```

Reply:
if this is important context, let's ensure it is in the git commit, not code comments - https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/